db/rmWrapper.py

```diff
@@ -52,6 +52,11 @@ def __ensure_columns_exist(self):
                 "table": "pokestop",
                 "column": "incident_expiration",
                 "ctype": "datetime NULL"
+            },
+            {
+                "table": "pokestop",
+                "column": "grunt_type",
+                "ctype": "tinyint(1) NULL"
```

Comment:
is tinyint enough here? if I remember right it has a value of max 127 - maybe it's overkill with smallint here, but who knows how many other types of character displays they might add in the future.